distributor/protocols/tdp/client.go

```diff
@@ -89,7 +88,7 @@ func (c *Conn) Block(_ context.Context, ref torus.BlockRef) ([]byte, error) {
 	ref.ToBytesBuf(c.buf[1:])
 	_, err := c.conn.Write(c.buf)
 	if err != nil {
-		fmt.Println("couldn't write")
+		clog.Errorf("couldn't write")
```

Comment:
Call side outputs in [tick.go#L86-L92](https://github.com/coreos/torus/blob/master/distributor/rebalance/tick.go#L86-L92).
But when error was caused by `i/o timeout` like below, it is difficult to find where the error returned like write "first 2 bytes" or "data". I updated this PR to distinguish the error.

~~~
....
couldn't write data
2016-12-11 13:50:57.362726 E | rebalance: couldn't rebalance block br 2 : 4 : 2: write tcp 127.0.0.1:42604->127.0.0.1:4000: i/o timeout
...
~~~